keps/sig-network/20190603-EndpointSlice-API.md

```diff
@@ -127,7 +127,15 @@ type AddressType string
 
 const (
     // AddressTypeIP represents an IP Address.
+    // This address type has been deprecated and has been replaced by the IPv4
+    // and IPv6 adddress types. New resources with this address type will be
+    // considered invalid. This will be fully removed in 1.18.
+    // +deprecated
```

Comment:
If we're promoting to Beta in v1.17 should we just remove this instead of deprecating it since it's still an alpha API?

Reply:
Yeah. This is sort of breaking change anyway.

Reply:
That would certainly be easier. Not sure the level of backwards compatibility we're obligated to provide. The risk seems to be that kube-proxy could potentially end up dropping all iptables/ipvs routes until the EndpointSlice controller created new and valid IPv4/IPv6 EndpointSlices that an updated version of kube-proxy recognized. As far as I can tell, kube-proxy does not currently filter by address type, so the 1.16 version of kube-proxy would likely be compatible with slices that had `IPv4` address types.